### ISSUE TITLE
Add filters on OOI overview

### ISSUE THREAD

**Issue by TwistMeister** [minvws/nl-kat-rocky#145]
Designs will be made and added by @NoemiOverdijk (ticket 142)

- [ ] As a user I want to filter the OOI overview list by OOI type, to narrow it down
- [ ] As a user I want to filter the OOI overview list on OOI scan profile type
- [ ] As a user I want to filter the OOI overview list on OOI scan profile level
- [ ] As a user I want to filter the OOI overview list on OOI origin type

**Comment by underdarknl** [comment]
Lets start with adding the Required Octopoes API options for this.

**Comment by TwistMeister** [comment]
@underdarknl Yes, for more extensive filter options that's indeed what we want, but these filter options are already present so after consulting @Lisser we figured that we could implement these already.

Do you suggest to wait on the new octopoes altogether so we don't have to refactor this by the time v3 is ready?
